libs/wingc/src/lib.rs

```diff
@@ -365,9 +368,24 @@ pub fn compile(
 		return Err(());
 	}
 
-	let mut jsifier = JSifier::new(&types, &files, app_name, &project_dir, true);
-	jsifier.jsify(&scope);
-	jsifier.emit_files(&out_dir);
+	let mut jsifier = JSifier::new(&mut types, &files, app_name, &project_dir, true);
+
+	// -- LIFTING PHASE --
+
+	let mut lift = LiftTransform::new(&jsifier);
```

Comment:
Not critical **but** I'd remove passing the `jsifier` into `LiftTransform` and have it create an internal jsifier. This will make it purer and make sure we don't mix the phases. Looking at the code I see that `LiftTrasform` never changes the jsifier's state it only changes `jsifier.types` which can be passed directly into `LiftTrasform` (the same way it's passed into `Jsifier::new` above.